lib/geoengineer/resources/aws/iam/aws_iam_saml_provider.rb

```diff
@@ -6,16 +6,28 @@
 class GeoEngineer::Resources::AwsIamSamlProvider < GeoEngineer::Resource
   validate -> { validate_required_attributes([:name, :saml_metadata_document]) }
 
-  after :initialize, -> { _terraform_id -> { name } }
+  after :initialize, -> {
+    _terraform_id = self[:name]
+  }
+
+  after :initialize, -> {
+    _geo_id self[:name]
```

Comment:
@grahamjenson would love your opinion on the way I am setting _geo_id and _terraform_id - it was very brittle and I could not get my local plan/apply to work without this configuration.